lib/std/math/big/int_test.zig

```diff
@@ -2671,6 +2671,87 @@ fn popCountTest(val: *const Managed, bit_count: usize, expected: usize) !void {
     try testing.expectEqual(expected, val.toConst().popCount(bit_count));
 }
 
+test "big int extractBits" {
```

Comment:
Some possible tests to have more confidence everything is correct:
- min + max supported sized type
- 2 and 3 limb sizes
- random input on your pc for varying (in CI with constant) seed (for both mask and original number): extract bit sequence, negate mask to extract the other bit sequence, applying or of extracted bit sequences must be identical to original.
- same principle: set random bit sequence, negate mask to set the other one, applying or must be identical to original or destination number. Not sure how to best create destination number.
- unclear and open (not feasible to solve with this PR): how to get structured testing of UUM for bigint and bigfloat

Random input can be taken like this:
```zig
const RndGen = std.rand.DefaultPrng;
    var rnd = RndGen.init(42);
    var i: u32 = 0;
    while (i < 10_000) : (i += 1) {
        var rand_num = rnd.random().int(i64);
        try test__popcountdi2(rand_num);
    }
```

Reply:
Yeah issue with creating the destination value is that we need to have a known-good implementation to generate it. I guess we could just write this in regular Zig though and converting to a bigint.

Reply:
I'm actually a bit skeptical of this now, since the actual opportunities for mistakes here would be around having multiple limbs. Generating random `u64`s likely wouldn't help us catch these, would it?

Reply:
Hmm. I think we should look at a more reasoned set of tests then. The current ones effectively just test the basic bitmask logic, it'd be good to add some more to ensure that the boundaries between limbs are handled correctly, and that a difference in sizes is handled correctly. I'm fairly confident in both of these, but definitely can't hurt to strengthen this.

Reply:
Uninitialised limbs aren't a worry though, since the output bigint is set to zero at the beginning.